lib/tasks/add_stake_pools.rake

```diff
@@ -32,13 +32,29 @@ stake_pools = [
 ]
 
 manager_fees = {
-  socean: 2,
+  socean: 0.16,
```

Comment:
Socean's manager fee is 2%, not 0.16. Why did you change that?
See this part on https://www.socean.fi/:
What fees does Socean charge?
We charge a one-time deposit fee (0.15% of transaction size), a one-time withdrawal fee (0.3% of transaction size), and an ongoing management fee (~2.0% of staking rewards). These fees may be subject to change. At 7.5% APY, it takes around 8 days to recoup the 0.15% deposit fee.